ansible/config/linux/docker_dnsmasq_conf

```diff
@@ -0,0 +1,19 @@
+# Set dns forwarder.
```

Comment:
All this stuff sound suspicious... have you followed the guide on our playbook? This stuff sound absolutely not needed to me...